src/ameba.ts

```diff
@@ -91,6 +91,11 @@ export class Ameba {
                             `[${issue.rule_name}] ${issue.message}`,
                             this.parseSeverity(issue.severity)
                         );
+                        diag.code = {
+                            value: "Docs",
+                            target: Uri.parse(`https://crystal-ameba.github.io/ameba/Ameba/Rule/${issue.rule_name}.html#overview`)
```

Comment:
this should probably use the versioning. @Sija wdyt?

Reply:
It's good for now, although I'd prefer to avoid 3rd party services.